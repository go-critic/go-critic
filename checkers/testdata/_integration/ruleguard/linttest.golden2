exit status 1
./f1.go:29:6: ruleguard: use errors.New(k()) or fmt.Errorf("%s", k()) instead
