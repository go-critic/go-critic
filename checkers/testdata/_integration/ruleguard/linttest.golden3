exit status 1
./f1.go:18:2: ruleguard: may be simplified to one if
./f1.go:12:10: ruleguard: regexp compilation should be avoided on the hot paths
./f1.go:29:6: ruleguard: use errors.New(k()) or fmt.Errorf("%s", k()) instead
