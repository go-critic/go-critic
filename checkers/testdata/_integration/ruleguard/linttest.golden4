exit status 1
./f1.go:8:1: ruleguard: error as an underlying type is probably a mistake
./f1.go:18:2: ruleguard: may be simplified to one if
./f1.go:12:36: ruleguard: suggestion: s1+s2
./f1.go:29:6: ruleguard: use errors.New(k()) or fmt.Errorf("%s", k()) instead
