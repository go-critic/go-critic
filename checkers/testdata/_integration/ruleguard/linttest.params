check -@ruleguard.rules ./rules.go -enable ruleguard -@ruleguard.disable=#test ./... | linttest.golden
check -@ruleguard.rules ./rules.go -enable ruleguard -@ruleguard.enable=#style -@ruleguard.disable=#test ./... | linttest.golden2
