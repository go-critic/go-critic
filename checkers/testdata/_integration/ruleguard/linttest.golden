exit status 1
./f1.go:5:1: ruleguard: error as an underlying type is probably a mistake
./f1.go:9:10: ruleguard: suggestion: s1+s2
./f1.go:25:6: ruleguard: use errors.New(k()) or fmt.Errorf("%s", k()) instead
